aiohue/v2/models/scene.py

```diff
@@ -72,7 +72,7 @@ class Scene:
     # actions: required(array of Action)
     # List of actions to be executed synchronously on recall
     actions: list[Action]
-    palette: PaletteFeature
+    palette: PaletteFeature | None
```

Comment:
please move it to the optional values a bit below and give it a default value of None